foursight_core/react/api/cookie_utils.py

```diff
@@ -21,6 +22,24 @@ def read_cookie(request: dict, cookie_name: str) -> Optional[str]:
     return cookie_morsel.value if cookie_morsel else None
 
 
+def read_cookie_bool(request: dict, cookie_name: str) -> bool:
+    return str_to_bool(read_cookie(request, cookie_name))
+
+
+def read_cookie_int(request: dict, cookie_name: str, fallback: int = 0) -> int:
```

Comment:
What's the difference between a fallback and a default?

Reply:
No difference, should I prefer "default" rather than "fallback" as a name?